client/app.jsx

```diff
@@ -8,8 +8,9 @@ class App extends React.Component {
     super(props);
 
     // this.requestedId = JSON.parse(window.location.href.split('/')[3]); 
-    this.requestedId = JSON.parse(window.location.pathname.slice(1));
+    this.requestedId = JSON.parse(window.location.pathname.slice(1, -1));
```

Comment:
Just refactored use this method. Although I didn't have to JSON.parse. Is that necessary?

Reply:
I seem to remember that I was doing this for my tests at them moment, but probably something that can be refactored later.  Only necessary if you need it.